### PR TITLE
doc: LP token guide

### PR DESCRIPTION
**Pull request by kupermind**
- Changing on-chain architecture image to reflect wrapped `veOLAS` contract;
- Adding LP enabling doc.

### CODE REVIEW

**Review by 77ph — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ For testing purposes, the hardhat node deployment script is located [here](https
 ## Deployed Protocol
 The list of addresses can be found [here](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/mainnet_addresses.json).
 
+## LP Token Guide
+In order to enable OLAS-based LP tokens, it is advised to check with the following [list of instructions](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/LP_token_guide.md).
```
Comment (mariapiamo) [inline README.md:93]:
```suggestion
It is advised to check the following [list of instructions](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/LP_token_guide.md) before enabling OLAS-based LP tokens. 
```

File: docs/LP_token_guide.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# LP Token Guide
+## UniswapV2
+To illustrate the mechanism, the well-known DAI / wETH pair is used: https://app.zerion.io/tokens/UNI-V2-0xa478c2975ab1ea89e8196811f51a7b7ade33eb11
+
+That LP token has the following address: `0xa478c2975ab1ea89e8196811f51a7b7ade33eb11`.
+
+- Get a description of what is inside the LP token pool, one token of which is [OLAS](https://etherscan.io/address/0x0001A500A6B18995B03f44bb040A5fFc28E45CB0).
+  In our example DAI goes instead of OLAS with the following address: `0x6B175474E89094C44Da98b954EedeAC495271d0F`.
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:8]:
```suggestion
  In our example, we are considering DAI in place of OLAS. DAI token has the following address: `0x6B175474E89094C44Da98b954EedeAC495271d0F`.
```

File: docs/LP_token_guide.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# LP Token Guide
+## UniswapV2
+To illustrate the mechanism, the well-known DAI / wETH pair is used: https://app.zerion.io/tokens/UNI-V2-0xa478c2975ab1ea89e8196811f51a7b7ade33eb11
+
+That LP token has the following address: `0xa478c2975ab1ea89e8196811f51a7b7ade33eb11`.
+
+- Get a description of what is inside the LP token pool, one token of which is [OLAS](https://etherscan.io/address/0x0001A500A6B18995B03f44bb040A5fFc28E45CB0).
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:7]:
```suggestion
- Get a description of what is inside the LP token pool, one of the tokens has to be [OLAS](https://etherscan.io/address/0x0001A500A6B18995B03f44bb040A5fFc28E45CB0).
```

File: docs/LP_token_guide.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  In our example DAI goes instead of OLAS with the following address: `0x6B175474E89094C44Da98b954EedeAC495271d0F`.
+- Open an ETH explorer in a chosen network (here the ETH mainnet is used): https://etherscan.io/token/0xa478c2975ab1ea89e8196811f51a7b7ade33eb11#readContract
+  - Read the output of methods `token0()`, `token1()`:
+    - `token0: 0x6B175474E89094C44Da98b954EedeAC495271d0F`,
+    - `token1: 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2`.
+    These addresses have to match with OLAS (DAI) and wETH addresses.
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:13]:
```suggestion
   In the example, these addresses are DAI and wETH addresses. In the case of OLAS-wETH LP tokens, these addresses have to match with OLAS (DAI) and wETH addresses.
```

File: docs/LP_token_guide.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+- Check the pool address:
+  - Verification logic (no need to do it every time):
+    - Open the Uniswap documentation and find its router address: https://docs.uniswap.org/contracts/v2/reference/smart-contracts/router-02.
+    - On that page, one can find the following text: `UniswapV2Router02 is deployed at 0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D address on the Ethereum mainnet, and the Ropsten, Rinkeby, Görli, and Kovan testnets. It was built from commit 6961711.`
+    - Open the contract reading page: https://etherscan.io/address/0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D#readContract.
+    - Read the `factory()` method that must return the following address: `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f`.
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:19]:
```suggestion
    - Read the `factory()` method and check that returns the following address: `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f`.
```

File: docs/LP_token_guide.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+  - Open the UniswapV2Factory contract: https://etherscan.io/address/0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f#readContract
+  - Call the `getPair()` method with the following input:
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:22]:
```suggestion
  - Call the `getPair()` method with the input `OLAS token address` and `wETH token address`. In our DAT-wETH examples, the inputs are the following:
  
```

File: docs/LP_token_guide.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    - `0x6B175474E89094C44Da98b954EedeAC495271d0F`,
+    - `0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2`.
+  - The output will be the following address: `0xA478c2975Ab1Ea89e8196811F51A7B7Ade33eB11`.
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:25]:
```suggestion
  - The output will be the LP token address. In this example, this is the following address: `0xA478c2975Ab1Ea89e8196811F51A7B7Ade33eB11`.
```

File: docs/LP_token_guide.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+  - If the address match with the given original address `0xa478c2975ab1ea89e8196811f51a7b7ade33eb11`, everything is correct.
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:26]:
```suggestion
  - If the address match with the given original LP-token address (in this example, `0xa478c2975ab1ea89e8196811f51a7b7ade33eb11` as well) everything is correct.
```

Comment (kupermind) [reply docs/LP_token_guide.md:26]: This is not exactly the same as the case is different in the output, but technically they are the same.

File: docs/LP_token_guide.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    - Open the contract reading page: https://etherscan.io/address/0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D#readContract.
+    - Read the `factory()` method and check that it returns the following address: `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f`.
+
+  - Open the UniswapV2Factory contract: https://etherscan.io/address/0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f#readContract
+  - Call the `getPair()` method with `OLAS token address` and `wETH token address` as inputs. In our DAT-wETH examples, the inputs are the following:
+    - `0x6B175474E89094C44Da98b954EedeAC495271d0F`,
+    - `0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2`.
+  - The output will be the LP token address. In this example, this is the following address: `0xA478c2975Ab1Ea89e8196811F51A7B7Ade33eB11`.
+  - If the address match with the given original LP-token address (in this example, `0xa478c2975ab1ea89e8196811f51a7b7ade33eb11`), everything is correct.
+- https://etherscan.io/address/0xa478c2975ab1ea89e8196811f51a7b7ade33eb11#internaltx
```
Comment (mariapiamo) [inline docs/LP_token_guide.md:27]:
```suggestion
```

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**